src/components/common-components/LargeTextBlock/LargeTextBlock.js

```diff
@@ -0,0 +1,50 @@
+import React, {useState} from 'react';
+import './LargeTextBlock.css';
+import Fade from 'react-reveal/Fade';
```

Comment:
ntipick: alphabetize imports, but that's not a huge deal and can be solved by adding a linter to the repo